src/components/HdPager.vue

```diff
@@ -27,6 +19,7 @@
         class="pager__items__track"
       >
         <div
+          v-if="modifier === 'wide'"
```

Comment:
Would be nice if this was a computed prop that makes use of the modifier enum `HdPagerModifierEnum.WIDE.

```suggestion
          v-if="isWide"
```

Reply:
Good point, pushed it in https://github.com/homeday-de/homeday-blocks/pull/1095/commits/948912baf2fd53e17d11e791881137a4b320e485